paimon-core/src/test/java/org/apache/paimon/table/AppendOnlyFileStoreTableTest.java

```diff
@@ -85,6 +85,30 @@ public void testBatchReadWrite() throws Exception {
                                 "2|21|201|binary|varbinary|mapKey:mapVal|multiset"));
     }
 
+    @Test
+    public void testBranchBatchReadWrite() throws Exception {
```

Comment:
Could we also verify the main branch is not affected?

Reply:
Ok, see it's verified in the `FileStoreTableTestBase`, I think it's enough. Please ignore this comment.